support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.jsx

```diff
@@ -89,7 +94,7 @@ function getGuardianWeeklyOfferCopy() {
 }
 
 const chooseImage = images =>
-  (countryGroupId === 'GBPCountries' || countryGroupId === 'EURCountries' || countryGroupId === 'International' ? images[0] : images[1]);
+  (countryGroupId === 'EURCountries' || countryGroupId === 'International' ? images[0] : images[1]);
```

Comment:
I'm not sure this function really helps make the code any clearer - I would remove or rethink it. Even more so now we aren't using it for the G Weekly product.